presto-kafka/src/main/java/io/prestosql/plugin/kafka/schema/file/FileTableDescriptionSupplierModule.java

```diff
@@ -16,6 +16,7 @@
 import com.google.inject.Binder;
```

Comment:
Please change commit message to:
```
Extract AvroDecoderModule
```

Also describe why you are doing this, where you can say that this allows to use different implementation of avro decoders etc.